src/converters/degiroConverterV3.ts

```diff
@@ -21,6 +22,12 @@ export class DeGiroConverterV3 extends AbstractConverter {
    * @inheritdoc
    */
   public processFileContents(input: string, successCallback: any, errorCallback: any): void {
+    // my exported file contained many broken records like this. It's easy to recover them. So, why not?!
```

Comment:
What exactly is broken about this?

Reply:
check the comment below. The record is split between two lines with many commas in the middle.